src/components/chat/list-item.tsx

```diff
@@ -92,29 +108,48 @@ export default function ChatListItem({
           />
         ) : (
           <div className="content">
+            {chat.msg.includes('**@' + user.name + '**') && (
+              <p>you've been mentionded</p>
+            )}
             <div
               className="text-with-markdown"
               dangerouslySetInnerHTML={{
-                __html: sanitizeHtml(emoji.emojify(marked(chat.msg)), {
-                  // allow any css class for `code` and `span`
-                  allowedClasses: {
-                    code: false,
-                    span: false,
+                __html: sanitizeHtml(
+                  emoji.emojify(
+                    marked(
+                      chat.msg.replace(
+                        '**@' + user.name + '**',
+                        `<mention>**@${user.name}**</mention>`,
+                      ),
+                    ),
+                  ),
+                  {
+                    // allow any css class for `code` and `span`
+                    allowedClasses: {
+                      code: false,
+                      span: false,
+                    },
+                    allowedTags: sanitizeHtml.defaults.allowedTags.concat([
```

Comment:
nice finding, i was looking for `sanitizeHtml.defaults.allowedTags` the other day :+1: